### PR TITLE
Beautified code display in readme :)

### PR DESCRIPTION
**Pull request by jkbecker**
Hi, great project!
I tried to follow the installation procedure and noticed that the command-line examples are sometimes hard to follow because they don't get printed as intended. I reformatted all the code sections, it looks pretty now :)

### CODE REVIEW

**Comment by jkbecker** [comment]
Sidenote, I just saw that #6 proposes the same thing... IMO mine looks a bit cleaner... but your choice.

**Comment by ghostop14** [comment]
Hey everyone!  I merged the readme updates.  I actually think I have a few more edits to make in the Pi section anyway.  The first round was on Raspbian squeeze and Stretch doesn't require the python build.  Thanks for the contribution!
